data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,454 @@ mission "Unfettered returning home"
+			action
+				set "unfettered: beaten the bugs"
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			not "unfettered: fled with the relic"
+		ship "Lightning Bug (Unfettered Shipyards)" "Dungeon Master"
+		personality launching marked staying uninterested
+		on provoke
+			dialog `Mera hails you. "Stop! I am only here as an observer. If you destroy my ship, the deal is off!"`
+		on kill
+			fail
+			set "unfettered: outcast"
+			dialog `As you destroy the Lightning Bug, along with Mera and her crew in it, you receive an angry hail from the Unfettered. "You completely disrespected the rules of a duel and killed one of our most prominent members! You will not be welcome here anymore!"`
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			has "unfettered: fled with the relic"
+		personality unconstrained nemesis launching heroic marked
+		ship "Sea Scorpion (Shredder)" "Hunter"
+	on visit
+		dialog "Either you have not defeated the Unfettered challenger in orbit yet, or you have broken the terms of the duel."
+	to complete
+		not "unfettered: fled with the relic"
+		# Needed to make sure the mission doesn't auto complete if no ships spawn.
+		has "unfettered: beaten the bugs"
+		has "flagship model: Anomalocaris"
+	on complete
+		clear "unfettered: beaten the bugs"
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an ancient Hai warship to beat an Unfettered challenge, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! I saw everything, and the data I got on the effects of these exotic weapons will be quite useful for my research. Was it hard for you?"`
+			choice
+				`	"It was fun."`
+					goto next
+				`	"This was easy!"`
+				`	"It was a challenge."`
+					goto challenge
+			`	"Don't get overconfident, Captain," she muses. "I am sure you know what happens to such pilots.`
+				goto next
+			label challenge
+			`	"Oh, so we almost got that ship after all? Maybe I should have participated in the fight after all.`
+			label next
+			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, right now, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us true Hai!"`
+			`	Mera then walks off as she begins reviewing data from your fight on a small device. For now, it seems that the Unfettered Hai have some sort of respect for you, so long as you have the Anomalocaris.`
+
+
+
+mission "Undeserved Anomalocaris"
+	minor
+	invisible
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "ship model: Anomalocaris"
+		"reputation: Hai (Unfettered)" <= -1000
+		not "Hai Legacy 2: active"
```

Comment:
```suggestion
		has "ship model: Anomalocaris"
		"reputation: Hai (Unfettered)" <= -1000
		not "Hai Legacy 2: active"
```
With these conditions, technically, if the Uhai are mad enough, this can happen before your duel mission, and you could no longer have the Anomalocaris by it. Is that intended?

Reply:
you get a rep reset to 0 when you enter uhai space so its fine
except if you somehow bring it back down to -1000